data-model/src/main/java/org/alfresco/service/cmr/search/SearchParameters.java

```diff
@@ -1641,6 +1647,19 @@ public void setIncludeMetadata(boolean includeMetadata)
     {
         this.includeMetadata = includeMetadata;
     }
-    
+
+    public int getTrackTotalHits()
+    {
+        return trackTotalHits;
+    }
+
+    /**
+     * Configure the limit to track the total hits on search results
```

Comment:
I still didn't get what the field is for - sorry. My guess is this:
```suggestion
     * Set a maximum value for the report of total hits. The reported number of hits will never exceed this limit even if more are found.
```